io.py

```diff
@@ -100,15 +100,33 @@ def read_bdi(paths, study='C', **kwargs):
 
     return bdi.set_index('ID')
 
+    if study == 'D':
+        df = pd.read_excel(op.join(beh_dir, 'subject_data.xlsx'))
+        if full_table:
+            bdi = pd.read_excel(op.join(beh_dir, 'subject_data.xlsx'))
+            sel_col = ['ID','MDD', 'sex', 'age', 'BDI'] # czy są potrzebne jeszcze inne kolumny?
+            sel_col = [col for col in sel_col if col in bdi.columns]
+            bdi = bdi[sel_col]
+        else:
+            bdi = pd.read_excel(op.join(beh_dir, 'subject_data.xlsx'), header=None,
+                                names=['id', 'MDD', 'BDI'])
+
+    return bdi.set_index('ID')
+
+
+
 
 def study_C_reformat_original_beh_table(df):
     '''Select and recode relevant columns from behavioral table.
 
     This function is not used in "Three times NO" paper.
     '''
     # select relevant columns
-    df = df[['ID', 'DATA BADANIA', 'WIEK', 'PŁEĆ', 'WYKSZTAŁCENIE',
-             'DIAGNOZA', 'BDI-II']]
+    sel_col = ['ID', 'DATA BADANIA', 'WIEK', 'PŁEĆ', 'WYKSZTAŁCENIE',
+               'DIAGNOZA', 'BDI-II']
+    sel_col = [col for col in sel_col if col in df.columns]
+    df = df[sel_col]
```

Comment:
dlaczego to zmieniasz?

Reply:
(już widzę - z jakiegoś powodu cofasz zmiany z moich ostatnich commitów, może źle rozwiązałaś merge conflict?)

Reply:
może, nie zauważyłam żeby git coś narzekał, mogę te zmiany, które nie są moje, cofnąć i zrobić nowy komit, będzie ok?